### ISSUE TITLE
Hackowanie gier w JS

### ISSUE THREAD

**Issue by BTMPL** [meetjspl/lodz#7]
Gry w JS - w jaki sposób możemy ułatwić sobie życie jako gracze oraz jak możemy utrudnić i wykryć manipulacje tworząc swoje aplikacje.

Poza suchą teorią przykład live-hacking gier prostych i zabezpieczonych.

**Comment by mostr** [comment]
Hej @BTMPL, planujemy spotkanie po wakacjach 28.09, czy temat prezentacji nadal aktualny i czy ten termin byłby ok?

**Comment by BTMPL** [comment]
Tak, myślę, że dało by radę przygotować coś co ma ręce i nogi - co prawda przykład, który miałem użyć już nie jest chyba online, ale mam kopię testową, więc można ugryźć.

Daj mi z 2 dni, to przygotuję jakiś brief i zbiorę trochę materiału

**Comment by mostr** [comment]
Dobra, to może inaczej. Zaklepałbym Twój temat jako pierwszy na październikowy meet.js, co? Będzie możliwosć żeby to na spokojnie przygotować, bo przed chwilą pojawiła się opcja prezentacji o Angular 2 (co w połączeniu z Ionic 2 robi się fajnie spójne). Chyba że wolisz zostać przy najbliższym spotkaniu, to wtedy nic nie zmieniamy i zaklepuję Cię na wrzesień jeszcze.

**Comment by BTMPL** [comment]
Ok, nie ma sprawy. Sam koniec października mogę być niedostępny (nie wiem ew. na kiedy planujesz), ale to będę wcześniej wiedział i dam znać.

**Comment by mostr** [comment]
Super, niedługo po wrześniowym zaczynamy organizować kolejny, więc parę dni poźniej powinniśmy znać już termin. Dzięki!

**Comment by mostr** [comment]
Hej, z tego co wiem to chyba u Ciebie październikowy meetup (31.10) odpada jeśli chodzi o termin? Czy coś się zmieniło i chcesz zrobić talka?

**Comment by BTMPL** [comment]
Hej; niestety wypada mi cały wcześniejszy weekend z kalendarza więc nie będę miał czasu się przygotować - ale chętnie na listopadowy :)

**Comment by BTMPL** [comment]
Hej - listopadowe sloty już zajęte? Jak tak, to tylko informuje, że  jakby co na grudzień/styczeń jestem wolny ;)

**Comment by mostr** [comment]
O masz :( Zapomniałem zupełnie o Twojej prezentacji, przepraszam :/ 
Tak, praktycznie już mamy całość dograną. Wrzucę Cie w takim razie wstępnie na następne spotkanie (grudzień/styczeń - zobaczymy). Jeszcze raz przepraszam za babola z mojej winy.

**Comment by BTMPL** [comment]
Spoko, nie ma problemu - więcej czasu na przygotowanie się :)
